gazelle_cabal/lang.go

```diff
@@ -205,11 +206,35 @@ func (*gazelleCabalLang) Fix(c *config.Config, f *rule.File) {
 	ruleInfos := cabalToRuleInfos(cabalFiles)
 
 	for _, r := range f.Rules {
+		print(r)
+
 		if !r.ShouldKeep() &&
 			isHaskellRule(r.Kind()) &&
 			!hasRuleInfo(ruleInfos, r.Kind(), r.Name()) {
 			r.Delete()
 		}
+
+		if !r.ShouldKeep() &&
+			r.Kind() == "stack_snapshot" {
+			print("Case accessed")
+			var list []string
+			pack := r.Attr("packages")
+			switch expr := pack.(type) {
```

Comment:
This code is probably better put in a function that extracts an array of strings from an attribute value. `gazelle_haskell_modules` has such a function.